[Marketplace Contribution] Akamai WAF - Content Pack Update (#22576)
demisto/content: Packs/Akamai_WAF/ReleaseNotes/2_0_2.md

#### Integrations ##### 
##### Akamai WAF ##### 
- %%

> Deleted four arguments in setting YAML file for command "akamai-get-papi-property-rule":
>     - name: operation
>     - name: path
>     - name: value
>     - name: value_to_json
> 
> Added one argument in setting YAML file for command "akamai-patch-papi-property-rule-generic"
>     - name: value_to_json
> 
> %%
> - Updated the Docker image to: *demisto/akamai:1.0.0.38691*.

Reviewer: ```suggestion
- Removed the *operation*, *path*, *value*, *value_to_json* arguments from the ***akamai-get-papi-property-rule*** command as they are not supported. 
- Added the argument *value_to_json* to command ***akamai-patch-papi-property-rule-generic***
- Updated the Docker image to: *demisto/akamai:1.0.0.38691*.
```
